### DISCUSSION TITLE
Manifest and Favicon has error in vercel but works well in local

### DISCUSSION THREAD

**Discussion by iam-yan** [vercel/vercel#4572] [Help]
Hi about favicon and manifest.
![image](https://user-images.githubusercontent.com/48427014/83724866-8e798580-a673-11ea-9d4b-e7d3fac30f80.png)
```
<meta name="viewport" content="width=device-width, initial-scale=1" />
          <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
          <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
          <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
          <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
          <link rel="manifest" href="/manifest.json" />
          <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
          <meta name="msapplication-TileColor" content="#603cba" />
          <meta name="theme-color" content="#ffffff" />
```

when I visited the deployed app I got an error:
`Manifest: Line: 1, column: 1, Syntax error.`
And favicon is not working...

But things works well in local development.
What have I missed...?

**Comment by Shubhampandeysp01** [comment]
I

> Hi about favicon and manifest. ![image](https://user-images.githubusercontent.com/48427014/83724866-8e798580-a673-11ea-9d4b-e7d3fac30f80.png)
> 
> ```
> <meta name="viewport" content="width=device-width, initial-scale=1" />
>           <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
>           <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
>           <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
>           <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
>           <link rel="manifest" href="/manifest.json" />
>           <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
>           <meta name="msapplication-TileColor" content="#603cba" />
>           <meta name="theme-color" content="#ffffff" />
> ```
> 
> when I visited the deployed app I got an error: `Manifest: Line: 1, column: 1, Syntax error.` And favicon is not working...
> 
> But things works well in local development. What have I missed...?

I'm facing the same issue. Any solution?
